table.csv

```diff
@@ -505,7 +506,7 @@ ssz-sha2-256-bmt,               multihash,      0xb502,         draft,      SSZ
 json-jcs,                       ipld,           0xb601,         draft,      The result of canonicalizing an input according to JCS - JSON Canonicalisation Scheme (RFC 8785)
 iscc,                           softhash,       0xcc01,         draft,      ISCC (International Standard Content Code) - similarity preserving hash
 zeroxcert-imprint-256,          zeroxcert,      0xce11,         draft,      0xcert Asset Imprint (root hash)
-varsig,                         varsig,         0xd000,         draft,      Namespace for all not yet standard signature algorithms
+nonstandard-sig,                varsig,         0xd000,         draft,      Namespace for all not yet standard signature algorithms
```

Comment:
```suggestion
nonstandard-sig,                varsig,         0xd000,    deprecated,      Namespace for all not yet standard signature algorithms
```

@vmx new status column type, indicating eventual removal?

Reply:
Works for me.

Reply:
Works for me too!